src/base/utils/net.cpp

```diff
@@ -114,6 +181,46 @@ namespace Utils
             return canonical;
         }
 
+        std::optional<std::pair<QString, QString>> parseIpRange(const QString &filterStr)
+        {
+            const QChar iprange_sep = u'-';
+            const QChar cidr_indicator = u'/';
+            QString first, last;
+            if (filterStr.contains(iprange_sep))
+            {
+                // ipfilter.dat's ip range format eg.
+                // "127.0.0.0 - 127.255.255.255"
```

Comment:
Hmm, this is not exactly true, `ipfilter.dat` uses slightly different format with mandatory leading zeroes.

https://github.com/qbittorrent/qBittorrent/blob/6ac0c5a8b82918120b7db9ec7ef6834ec9ea240d/src/base/bittorrent/filterparserthread.cpp#L201-L202

Maybe better to just remove it's mention.

```suggestion
                // ip range format eg.
                // "127.0.0.0 - 127.255.255.255"
```